serenata_toolbox/serenata-toolbox

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+
+import os
+
+from argparse import ArgumentParser
+
+from serenata_toolbox import log
+from serenata_toolbox.chamber_of_deputies.reimbursements import Reimbursements as ChamberDataset
+from serenata_toolbox.datasets import Datasets
+from serenata_toolbox.federal_senate.dataset import Dataset as SenateDataset
+
+
+def parser():
+    parser = ArgumentParser(description='Support Serenata de Amor and Rosie development.',
+                            usage='%(prog)s path [options]')
+    parser.add_argument('path', nargs='?', default='data/', help='directory for storing the datsets')
+    parser.add_argument('--dataset', '-d', nargs='+', help='the desired dataset to be downloaded')
+    parser.add_argument('--year', '-y', nargs='+', help='narrow the download to a list of years')
+    return parser.parse_args()
+
+
+def main():
+    args = parser()
+    path = args.path
+    if not os.path.isdir(path):
+        try:
+            os.mkdir(os.path.join(path))
+        except OSError:
+            log.error('Couldn\'t create the output directory')
+            raise
+    if not args.dataset:
+        datasets = Datasets(path)
+        datasets.downloader.download(datasets.downloader.LATEST)
+        return
+    if 'reimbursements' in args.dataset:
+        if args.year:
+            chamber = ChamberDataset(path, args.year)
+        else:
+            chamber = ChamberDataset(path)
```

Comment:
I guess if the user doesn't pass `--year` it becomes `None`. So there's no problem in calling `ChamberDataset(path, args.year)` in both cases. But this is merely a guess that needs to be checked.

Reply:
Hum, I think I got some error when I tried that. Let me try again.

Reply:
So maybe `chamber = ChamberDataset(path, args.year or None)`